VmAgent.Core.UnitTests/VmConfigurationTests.cs

```diff
@@ -98,19 +98,50 @@ public void VmStartupScriptEnvVariablesWithPorts()
             }
         }
 
+        [TestMethod]
+        [TestCategory("BVT")]
+        public void EnvironmentVariablesWithMultipleIpAddresses()
+        {
+            SessionHostsStartInfo sessionHostsStartInfo = CreateSessionHostStartInfo(new Dictionary<string, string>());
+            sessionHostsStartInfo.PublicIpAddresses = new List<PublicIpAddress>()
+            {
+                new PublicIpAddress()
+                {
+                    IpAddress = "127.0.0.1",
+                    FQDN = "myFQDN",
+                    RoutingType = RoutingType.Internet
+                },
+                new PublicIpAddress()
+                {
+                    IpAddress = "127.0.0.2",
+                    FQDN = "myFQDN2",
+                    RoutingType = RoutingType.Microsoft
+                },
+            };
+
+            IDictionary<string, string> envVariables = VmConfiguration.GetCommonEnvironmentVariables(sessionHostsStartInfo, VmConfiguration);
+            envVariables.Should().Contain("PF_PUBLIC_IP_ADDRESSES_COUNT", "2");
```

Comment:
I think we can make a const string for PF_PUBLIC_IP_ADDRESSES and create a helper function where tests env variables?

Reply:
I already did that in code. Personally, I don't like to mix the usage of test and src code. If there is a code change, I want to see that change in tests as well.

Reply:
yeah, I agree with Jota, let's leave it like this since tests sometimes are used as documentation so it's great to have the entire string here.